microsoft-edge/devtools-guide-chromium/performance/reference.md

```diff
@@ -0,0 +1,1238 @@
+
+To learn how to turn on screenshots, see [Capture screenshots while recording](#capture-screenshots-while-recording), above.
+
+To view a screenshot of how the page looked during that moment of the recording, hover over the **Overview**.  The **Overview** is the section that contains the **CPU** and **NET** charts:
+
+![Viewing a screenshot](./reference-images/screenshots-hover.png)
+
+You can also view screenshots by selecting a frame in the **Frames** section.  DevTools displays a small version of the screenshot in the **Summary** tab:
+
+![Viewing a screenshot in the Summary tab](./reference-images/summary-preview.png)
+
+To zoom in on the screenshot, click the thumbnail in the **Summary** tab.
+
+
+<!-- ------------------------------ -->
+#### View layers information
+<!-- https://developer.chrome.com/docs/devtools/performance/reference#layers -->
+
+To view advanced layers information about a frame:
+
+1. In the **Performance** tool, click the **Capture settings** (![Capture settings](./reference-images/capture-settings-icon.png)) button, and then select the **Enable advanced rendering instrumentation (slow)** checkbox.
+
+1. In the **Frames** section, select a frame.  DevTools displays information about the layers in the **Layers** tab at the bottom of the **Performance** tool:
+
+   ![The Layers pane](./reference-images/layers-all.png)
+
+The **Layers** tab works like the **Composited Layers** tab in the **3D View** tool.  To learn how to interact with the **Layers** tab, see [Navigate webpage layers, z-index, and DOM using the 3D View tool](../3d-view/index.md).
+
+
+<!-- ------------------------------ -->
+#### View paint profiler
+<!-- https://developer.chrome.com/docs/devtools/performance/reference#paint-profiler -->
+
+To view advanced information about a paint event:
+
+1. In the **Performance** tool, click the **Capture settings** (![Capture settings](./reference-images/capture-settings-icon.png)) button, and then select the **Enable advanced rendering instrumentation (slow)** checkbox.
+
+1. Select a **Paint** event in the **Main** section.  DevTools displays information about the paint event in the **Paint Profiler** tab:
+
+   ![The Paint Profiler tab](./reference-images/paint-profiler.png)
+
+
+<!-- ------------------------------ -->
+#### View CSS selector statistics
+<!-- section in downstream doc only -->
+
+To view statistics about the selectors of the CSS rules that got recalculated during a performance recording:
+
+*  In the **Performance** tool, click the **Capture settings** (![Capture settings](./reference-images/capture-settings-icon.png)) button, and then select the **Enable advanced rendering instrumentation (slow)** checkbox.
+
+   DevTools displays aggregate information about the selectors of the CSS rules that were recalculated during the recording in the **Selector Stats** tab:
+
+   ![The Selector Stats tab](./reference-images/selector-stats.png)
```

Comment:
Broken image here too.

Reply:
@captainbrosset  pls try:

1. go to internal preview anchor:
https://review.learn.microsoft.com/microsoft-edge/devtools-guide-chromium/performance/reference?branch=pr-en-us-3378#view-css-selector-statistics
2. In Edge, **F12** (DevTools) > Right-click **Refresh** > **Empty cache and hard refresh** > **F12**.

Predict the pngs are rendered.
 
Expect that to work, b/c the pngs are rendered at the below, and it is a frequent behavior/ action required.

external preview anchor:
https://github.com/MicrosoftDocs/edge-developer/blob/user/mikehoffms/perf-sync/microsoft-edge/devtools-guide-chromium/performance/reference.md#view-css-selector-statistics